mkdocs.yml

```diff
@@ -57,6 +56,7 @@ nav:
       - Release notes: test-suite/release-notes.md
     - Known issues and workarounds:
       - v2023.06: known_issues/eessi-2023.06.md
+  - API documentation: api/
```

Comment:
I would put this under `Advanced usage => Test suite`. I'm not sure if we should keep the test suite docs there indefinitely, might be better to have a separate `test suite` top level item. But that's of later concern. At least we should keep all the test suite stuff under the `Test suite:` list.

Reply:
Probably put it underneath `Release notes`